### DISCUSSION TITLE
How to use DeepMD to simulate monolayer adsorption and stabilize both the original and adsorbed structures

### DISCUSSION THREAD

**Discussion by Mingyu-Lin98** [deepmodeling/deepmd-kit#4862] [Q&A]
Dear developers
I have a question about how to set up a DeepMD simulation for the adsorption of a 2D material monolayer.
Our simulation workflow is as follows: after the DLP training process is complete, we use DLP to perform a minimization (Figure 1). We then use the final minimized structure to perform an NPT simulation (Figures 2 and 3).
However, when I use DP-Lammps to run the NPT simulation for the monolayer adsorption, the structure becomes unstable (it doesn't hold the whole surface), as shown in Figure 3.

My training set :
1.Substrate 50,300K (B4N4,B7N5,Si2BN,P-BN2,B4N,T-BN)
2.Metal Lithium 300K (Adsorption Atom)
3.Dimer (Contain Substrate and Adsorption Atom)
4.Full of Li Na Adsorption on Substrate(One&Both side )
5.Two of Li adsorption on Substrate(Only one side)

Descriptor Settings
Descriptor_type: se_a
rcut: 6.000000000001
rcut_smth: 0.5000000001

Loss Function Weights
start_pref_f : 1000
limit_pref_f : 5
start_pref_v : 10
limit_pref_v : 0.01
start_pref_e : 0.01
limit_pref_e : 1
<img width="893" height="680" alt="image" src="https://github.com/user-attachments/assets/262ad76b-18ae-4c24-ad31-880104e3c540" />
Fig.1 Minimize data(The final step by Na adsorption T-BN structure)
<img width="793" height="664" alt="image" src="https://github.com/user-attachments/assets/dc0bc6ef-1d13-42c7-9b9f-b04a4cb7a810" />
Fig.2 Npt simulation(before collapse, but the structure is already slightly distorted)
<img width="838" height="685" alt="image" src="https://github.com/user-attachments/assets/6aec54c0-1f28-48a6-86cf-2b263b131d3d" />
Fig.3 Npt simulation(Crash)
